lfs/scanner.go

```diff
@@ -187,6 +195,12 @@ func revListShas(refLeft, refRight string, opt ScanRefsOptions) (chan string, er
 		if refRight != "" && !z40.MatchString(refRight) {
 			refArgs = append(refArgs, refRight)
 		}
+	case ScanAll:
+		refArgs = append(refArgs, "--all")
+	case ScanDeltaToRemotes:
+		refArgs = append(refArgs, refLeft, "--not", "--remotes")
```

Comment:
ok, here is the change which makes use of current remote only: https://github.com/github/git-lfs/pull/578
I'll be happy with either of those 2.